src/main/java/emissary/config/ConfigUtil.java

```diff
@@ -74,28 +75,35 @@ public class ConfigUtil {
     public static final String PROJECT_BASE_ENV = "PROJECT_BASE";
 
     /** The package name where config stuff may be found */
+    @Nullable
```

Comment:
The fact that we need to put a `@Nullable` annotation on fields that we explicitly initialize as `null`